src/helix-ui/styles/modules/text-input.less

```diff
@@ -1,42 +1,28 @@
+@import (reference) "_utils";
+
 .hxTextCtrl {
+  #Util.placeholder();
+
   background-color: @gray-0;
   border: 1px solid @gray-500;
   border-radius: 2px;
-  color: @gray-800;
+  color: @gray-900;
```

Comment:
Updated for consistent styling. It was kinda obvious given that we have both a search box and text inputs in the same general location on the Search Box demo.

Reply:
is this reflected/updated in the design spec?

Reply:
It is not. I'll be sure to make a story to get the redlines updated, especially given that they're missing "disabled" specs, too.

Reply:
This is actually pending a bowling discussion. Will update tomorrow.